model_bakery/baker.py

```diff
@@ -493,9 +494,9 @@ def generate_value(self, field, commit=True):
         elif getattr(field, "choices"):
             generator = random_gen.gen_from_choices(field.choices)
         elif isinstance(field, ForeignKey) and issubclass(
-            self._remote_field(field).model, ContentType
+            self._remote_field(field).model, contenttypes.models.ContentType
```

Comment:
This isn't required but what do you think of extracting this condition into a variable or a method like `is_content_type` or something similar? In my opinion would be better to understand what is going on here. What do you think?

Reply:
Totally agree :+1: 

It was fixed by 6c90432